bakerydemo/base/models.py

```diff
@@ -159,14 +159,126 @@ class HomePage(Page):
         related_name='+',
         help_text='Homepage image'
     )
+    hero_text = models.CharField(
+        max_length=255,
+        help_text='Write an introduction for the bakery'
+        )
+    hero_cta = models.CharField(
+        verbose_name='Hero CTA',
```

Comment:
Ah, maybe that's an English-English expression? It stand for "call-to-action". Happy to spell it out more clearly in the verbose_name. Unless you think there might be a better, more generic English expression?